server/app/assets/javascripts/sort_selector.ts

```diff
@@ -0,0 +1,50 @@
+/** Responsible for controlling the sorting mechanism */
+export function sortSelectorElements(
+  sortSelectId: string,
+  sortElementId: string,
```

Comment:
I'm having some trouble understanding these param names and what this method does:
1. do the params all need to start with "sort"? Seems kind of redundant
2. sortElementId and sortSublistId are not IDs - querySelectorAll supports any selector and the use case in this PR is using a class selector. Would "elementSelector" and "sublistSelector" make more sense?
3. It seems like the hierarchy goes sublistSelector -> elementSelector, so would it make more sense to put them in that order?
4. Maybe add a jsdoc explanation for each param and what this method does in general? Based on the method name I first thought it would sort a <select> element but it's much too complicated for that.

Reply:
good suggestions! updated the list, param ordering and descriptions to be clearer based on your comments.